### PR TITLE
Blog post for announcing speakers WTD AU 2022

### PR DESCRIPTION
**Pull request by swapnilogale**
----
:books: Documentation preview :books:: https://writethedocs-www--1826.org.readthedocs.build/en/1826/

### CODE REVIEW

**Comment by swapnilogale** [comment]
Thanks for setting up the t-shirts pop up store @thatdocslady!

**Comment by swapnilogale** [comment]
@plaindocs could you review this? Hoping to get this out so I can work on the schedule next.

File: docs/_data/australia-2022-schedule.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+talks_day1:
```
Comment (plaindocs) [inline docs/_data/australia-2022-schedule.yaml:1]: This is all provisional right, as we're not publishing it?

File: docs/conf/australia/2022/news/announcing-speakers.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+You can apply until **{{ grants.ends }}** on `our website <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/opportunity-grants/>`_.
+
+Thanks to our sponsors
+----------------------
+
+We are so grateful to have our sponsors help in bringing these events to life every year. Thanks sincerely to the following companies for supporting the Write the Docs community:
+
+.. datatemplate::
+   :source: /_data/{{shortcode}}-{{year}}-config.yaml
+   :template: {{year}}/sponsors-simplelist.rst
```
Comment (plaindocs) [inline docs/conf/australia/2022/news/announcing-speakers.rst:61]: There are currently no sponsors, right?

**Review by plaindocs — APPROVED** [review]
LGTM

File: docs/_data/australia-2022-schedule.yaml (hunk 1 above)
Comment (swapnilogale) [reply docs/_data/australia-2022-schedule.yaml:1]: Yep

File: docs/conf/australia/2022/news/announcing-speakers.rst (hunk 2 above)
Comment (swapnilogale) [reply docs/conf/australia/2022/news/announcing-speakers.rst:61]: Yep. Still confirming

**Pull request merged**